src/Network/Network/Generated/LoadBalancer/FrontendIPConfiguration/AddAzureRmLoadBalancerFrontendIpConfigCommand.cs

```diff
@@ -61,6 +61,22 @@ public partial class AddAzureRmLoadBalancerFrontendIpConfigCommand : NetworkBase
             ValueFromPipelineByPropertyName = true)]
         public string PrivateIpAddress { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ParameterSetName = "SetByResourceSubnet",
+            HelpMessage = "The private IP address version of the IP configuration.",
+            ValueFromPipelineByPropertyName = true)]
+        [Parameter(
+            Mandatory = false,
+            ParameterSetName = "SetByResourceIdSubnet",
+            HelpMessage = "The private IP address version of the IP configuration.",
+            ValueFromPipelineByPropertyName = true)]
+        [ValidateSet(
+            MNM.IPVersion.IPv4,
```

Comment:
Missing import (`using MNM = Microsoft.Azure.Management.Network.Models;`) is breaking build. Same thing in `New` and `Set`